packages/config/config/devices/0x0086/templates/aeotec_template.json

```
@@ -0,0 +1,1132 @@
+				"value": 6
+			},
+			{
+				"label": "Home security",
+				"value": 7
+			},
+			{
+				"label": "Power management",
+				"value": 8
+			},
+			{
+				"label": "System notification",
+				"value": 9
+			},
+			{
+				"label": "Emergency alarm notification",
+				"value": 10
+			},
+			{
+				"label": "Clock notification",
+				"value": 11
+			}
+		]
+	},
+	"motion_report_type": {
+		"label": "Motion Sensor Triggered Command",
+		"valueSize": 1,
+		"minValue": 1,
+		"maxValue": 2,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Send Basic Set CC",
+				"value": 1
+			},
+			{
+				"label": "Send Sensor Binary Report CC",
+				"value": 2
+			}
+		]
+	},
+	"sensor_report_type": {
+		"label": "Motion Sensor Triggered Command",
+		"valueSize": 4,
+		"minValue": 16,
+		"maxValue": 256,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Send Sensor Binary Report CC",
+				"value": 16
+			},
+			{
+				"label": "Send Basic Set CC",
+				"value": 256
+			}
+		]
+	},
+	"trigger_off": {
+		"label": "Trigger Off Period",
+		"description": "Period after which to send Trigger Off command",
+		"valueSize": 2,
+		"unit": "seconds",
+		"minValue": 1,
+		"maxValue": 15300,
+		"defaultValue": 240,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"selective_reporting": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Selective Reporting",
+		"description": "Report only when the threshold defined below is exceeded"
+	},
+	"lock_configuration": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Lock Configuration"
+	},
+	"low_battery_button": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Wireless Button Low Battery Report"
+	},
+	"low_temp_alarm": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Low Temperature Alarm (-15°C)"
+	},
+	"report_interval": {
+		"valueSize": 4,
+		"unit": "seconds",
+		"minValue": 1,
+		"maxValue": 65535,
+		"defaultValue": 720,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"factory_reset": {
+		"label": "Reset to Factory Defaults",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Normal Operation",
+				"value": 0
+			},
+			{
+				"label": "Reset to Factory Default",
+				"value": 1
+			}
+		]
+	},
+	"factory_reset_exclude": {
+		"label": "Reset to Default Factory Setting",
+		"description": "This parameter is used to reset product to default factory setting. (there are just 2 values can be used)",
```

```suggestion
		"description": "This parameter is used to reset the product to default factory setting.",
```
Also, isn't it called "factory default"?